### ISSUE TITLE
How to use ReCaptcha in blueprint

### ISSUE THREAD

**Issue by paulocauca** [mardix/flask-recaptcha#19]
Hi Guys, 
Sorry to open with a issue this question. 

Can you help me implement the reCaptcha with blueprint ?

__init__.py
`recaptcha = ReCaptcha()
recaptcha.init_app(app)`

html file

`{{ recaptcha }}`

how can I call recaptcha to verify in login.py 

Thank you

**Comment by KnugiHK** [comment]
I did that by assigning the instance of ReCaptcha() in another module. Let's say

app.py (main)
```python
import global # that's the "another" module
-------
global.recaptcha = ReCaptcha(app)
-------
@app.route("/"):
def index():
    return render_template("index.html", recaptcha=global.recaptcha.get_code())
```
html file
```
{{recaptcha}}
```
login.py
```python
@login.route("/verify"):
def verify():
    if global.recaptcha.verify():
        -------
```

**Comment by paulocauca** [comment]
Hi, Thanks , this global is a module that I install with pip ?

**Comment by KnugiHK** [comment]
No, the module "global" is just another python module file, you can rename to whatever you want.
E.g
variable.py #the global one
app.py
login.py

Another example:
glo.py #the global one
app.py
login.py
